jsp/pom.xml

```diff
@@ -76,9 +76,9 @@
             <version>${project.parent.version}</version>
         </dependency>
         <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>servlet</artifactId>
-            <version>${project.parent.version}</version>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>tck-runtime</artifactId>
```

Comment:
Is this artifact available in maven staging or central yet ? I cant seem to locate it.

Reply:
Both a SNAPSHOT and M1 are in staging (they are different releases but should be functionally identical)
https://jakarta.oss.sonatype.org/content/groups/staging/jakarta/servlet/